liblouis/lou_translateString.c

```diff
@@ -62,18 +62,28 @@ allocStringBuffer(int index, int length) {
 	return _lou_allocMem(alloc_passbuf, index, 0, length);
 }
 
-static widechar *stringBuffers[MAXPASSBUF] = { NULL };
-static int stringBuffersInUse[MAXPASSBUF] = { 0 };
-static StringBufferPool stringBufferPool = (StringBufferPool){ MAXPASSBUF, stringBuffers,
-	stringBuffersInUse, &allocStringBuffer, NULL };
+static const StringBufferPool *stringBufferPool = NULL;
+
+static void
```

Comment:
Why was this needed? The previous code looks good enough to me. Why is a function needed? And what does this change have to do with Travis?

Reply:
Previous code built fine on my machine but Travis complained.

Reply:
Weird, maybe travis is using gcc and you're using clang?

Reply:
I'm using gcc.